resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,74 @@
+@extends('auth.layouts.master')
+@section('content')
+    <div class="container-fluid px-1 px-md-5 px-lg-1 px-xl-5 py-5 mx-auto">
+        <div class="d-flex justify-content-end language">
+            <ul>
+                <li class="{{app()->getLocale()=='en'?'active':''}}">
+                    <a href="{{route('change-language','en')}}">
+                        <img src="{{asset('images/us.png')}}" alt="English">
+                    </a>
+                </li>
+                <li class="{{app()->getLocale()=='vi'?'active':''}}">
```

Comment:
cách ra nhé, tương tự các chỗ khác
```suggestion
                <li class="{{ app()->getLocale() == 'vi' ? 'active' : '' }}">
```